flexmeasures/data/schemas/tests/test_scheduling.py

```diff
@@ -0,0 +1,78 @@
+from flexmeasures.data.schemas.scheduling.shiftable_load import (
+    ShiftableLoadFlexModelSchema,
+    LoadType,
+)
+
+from datetime import datetime
+import pytz
+
+
+def test_shiftable_load_flex_model_load(db, app, setup_dummy_sensors):
+
+    sensor1, _ = setup_dummy_sensors
+
+    schema = ShiftableLoadFlexModelSchema(
+        sensor=sensor1,
+        start=datetime(2023, 1, 1, tzinfo=pytz.UTC),
+        end=datetime(2023, 1, 2, tzinfo=pytz.UTC),
+    )
+
+    shiftable_load_flex_model = schema.load(
+        {
+            "consumption-price-sensor": sensor1.id,
```

Comment:
Move this field to flex context (as you did in `test_shiftable_load.py` or leave away if not needed.